tests/epics/signal/test_common.py

```diff
@@ -45,3 +45,41 @@ class MyEnum(StrictEnum):
     assert len(supported_vals) == 2
     assert "test_1" in supported_vals
     assert "test_2" in supported_vals
+
+
+def test_given_supersetenum_that_partial_matches_are_valid():
+    class MyEnum(SupersetEnum):
+        TEST_1 = "test_1"
+        TEST_2 = "test_2"
+
+    supported_vals = get_supported_values("", MyEnum, (MyEnum.TEST_1,))
```

Comment:
We should be testing the string values lookup here
```suggestion
    supported_vals = get_supported_values("", MyEnum, ("test_1",))
```